pkg/networkaware/util/util.go

```diff
@@ -63,12 +63,11 @@ func GetNodeRegion(node *v1.Node) string {
 		return ""
 	}
 
-	zone, _ := labels[v1.LabelTopologyRegion]
-	if zone == "" {
-		return ""
+	if region, ok := labels[v1.LabelTopologyRegion]; ok {
+		return region
 	}
 
-	return zone
+	return ""
```

Comment:
you can just:
```go
return labels[v1.LabelTopologyRegion]
```
if `v1.LabelTopologyRegion` is not in map it will return default value for string which is `""`

Reply:
+1

Reply:
Thanks, it has been modified